### ISSUE TITLE
Mirror iridiumbrowser repository to GitHub

### ISSUE THREAD

**Issue by longsleep** [iridium-browser/iridium-browser#14]
It would be very nice to have the Git source for Iridium available on GitHub. This allows a much simpler way to track merge and review of patches.

Any suggestions? Right now there are two problems - repository commit pack size limit and file size limits.

**Comment by fancycode** [comment]
We could avoid file size limits by just removing the large files. From what I see it's only one test data file which is above the 100MB limit. This is what I'm doing for the iridium-browser-ubuntu repository.

**Comment by longsleep** [comment]
How does this work with mirroring?

**Comment by fancycode** [comment]
Agreed, it's technically not mirrored that way, but we would have the code on GitHub and could fork, create pull-requests and comment on them.

Any changes we want to have committed would still have to be provided as patch and committed manually on `git.iridiumbrowser.de`.

**Comment by fancycode** [comment]
Mirroring on GitLab would work:
https://gitlab.com/fancycode/iridium-browser
